.env.test

```diff
@@ -1,6 +1,8 @@
 DEBUG=true
 SECRET_KEY='tests'
 DATABASE_URL='postgres://localhost/opencraft'
+INSTANCE_MYSQL_URL='mysql://root@localhost'
+INSTANCE_MONGO_URL='mongodb://localhost'
```

Comment:
Using `localhost` here is a bit confusing, since it will provision the databases on the machine that runs the instance manager, but then tell the instances the database is at `localhost`, i.e. on the host of the instances themselves.  These configuration options will have to point to a hostname that resolves from both the instance manager and the instances to the same database server (though `localhost` might work fine for testing).